docs/reference/cli-reference.md

```
@@ -212,6 +213,258 @@ phlo services logs -f dagster-webserver
+phlo plugin info dagster
+
+# Specify type
+phlo plugin info rest_api --type sources
+
+# JSON output
+phlo plugin info dagster --json
+```
+
+**Output**:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)